Sources/Plasma/PubUtilLib/plDrawable/plWaveSet7.cpp

```diff
@@ -2271,7 +2266,10 @@ void plWaveSet7::IAddShoreVertexShader(hsGMaterial* mat)
 
         vShader->SetVector(plShoreVS::kSinConsts, 1.f, -1.f/6.f, 1.f/120.f, -1.f/5040.f);
         vShader->SetVector(plShoreVS::kCosConsts, 1.f, -1.f/2.f, 1.f/24.f, -1.f/720.f);
-        vShader->SetVector(plShoreVS::kPiConsts, 1.f / (8.f*M_PI*4.f*4.f), M_PI/2.f, M_PI, M_PI*2.f);
+        vShader->SetVector(plShoreVS::kPiConsts, 1.f / (8.f*hsConstants::pi<float>*4.f*4.f),
+                                                 hsConstants::half_pi<float>,
+                                                 hsConstants::pi<float>,
+                                                 hsConstants::two_pi<float>);
```

Comment:
Strange indentation

Reply:
I aligned on the values (that is, the second constant is aligned with the start of the first constant), not the parameters...